<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PC" ignored="0" name="Test" tests="1" failures="1" timestamp="05 Jun 2020 21:35:30 GMT" time="20.465" errors="0">
  <testcase name="feature" time="18.934" classname="cucumberoptions.TestRunner">
    <failure type="java.lang.AssertionError" message="account already exist">
      <![CDATA[java.lang.AssertionError: account already exist
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertFalse(Assert.java:64)
at pageObjects.automationpracticePage.SignIn(automationpracticePage.java:102)
at stepDefinations.automationparcRegisteration.click_on_sign_in_button(automationparcRegisteration.java:22)
at âœ½.Click on Sign In2 Button(file:///C:/selenium/mavenproj/src/test/java/features/BDDassignment2.feature:6)
]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Test -->
